libraries/common-arm/src/sd.rs

```diff
@@ -1,14 +1,33 @@
 use core::marker::PhantomData;
```

Comment:
Some linter fixes which I missed in Noah's PR

Reply:
I missed these, thank you for fixing.